CONTRIBUTING.md

```diff
@@ -25,8 +25,16 @@ guidance below.
 ## Changing the code-base
 
 Generally speaking, you should fork this repository, make changes in your
-own fork, and then submit a pull-request. All new code should have associated unit
```

Comment:
I think for external contributions, we definitely want a fork. For internal (CFPB) contributors, I think there's a lot more fuzz. We have branching and merging guidelines in our working "building for cf.gov" documentation.